src/FSharpVSPowerTools/FSharpVSPowerTools.csproj

```diff
@@ -46,7 +46,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <DeployExtension>True</DeployExtension>
+    <DeployExtension>False</DeployExtension>
```

Comment:
Why do you need to turn this off? It no longer allows deploying the extension, which is quite counter-intuitive.